ev3dev/auto.py

```diff
@@ -8,11 +8,17 @@ def current_platform():
         return 'ev3'
     elif machine == 'armv6l':
         return 'brickpi'
+    elif machine == 'armv7l':
```

Comment:
True :(  Looks like I need to wait for https://github.com/ev3dev/ev3dev/issues/523 and then revisit auto.py